src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, import_prefix, client_name):
         ResourceType.MGMT_RESOURCE_FEATURES: '2015-12-01',
         ResourceType.MGMT_RESOURCE_LINKS: '2016-09-01',
         ResourceType.MGMT_RESOURCE_LOCKS: '2016-09-01',
-        ResourceType.MGMT_RESOURCE_POLICY: '2016-04-01',
+        ResourceType.MGMT_RESOURCE_POLICY: '2016-12-01',
```

Comment:
Does this change require an API version bump?

Reply:
Yes, unfortunately policy parameters weren't included in the 2016-04-01 PolicyDefinition.

[2016-04-01](https://docs.microsoft.com/en-us/python/api/azure.mgmt.resource.policy.v2016_04_01.models.policydefinition?view=azure-python)
vs
[2016-12-01](https://docs.microsoft.com/en-us/python/api/azure.mgmt.resource.policy.v2016_12_01.models.policydefinition?view=azure-python)

Reply:
In that case, you need to add an annotation to your params file so that the policy parameters don't appear when using the older API version (i.e. the stack profile).  It will look something like:

```Python
with VersionConstraint(ResourceType.MGMT_RESOURCE_POLICY, min_api='2016-12-01') as c:
   c.register_cli_argument('...', 'policy', ...)
```
Also,  recommend @derekbekoe take a look from the versioning perspective.